test/lib/ansible_test/_data/sanity/code-smell/runtime-metadata.py

```diff
@@ -68,7 +94,34 @@ def any_value(value):
     return value
 
 
-def validate_metadata_file(path, is_ansible):
+def get_ansible_version():
+    """Return current ansible-core version"""
+    from ansible.release import __version__
+
+    return LooseVersion(__version__)
```

Comment:
The `rc1`/`.dev0` could maybe be removed for clarity - not that 2.11.0 < 2.11.0.dev0 would really be a problem since there should be a deprecation period in ansible-core.

Reply:
Good idea! I took the corresponding code from validate-modules which removes the suffix.